jetbrains-rider/src/software/aws/toolkits/jetbrains/ui/wizard/DotNetSamProjectGenerator.kt

```diff
@@ -56,7 +57,7 @@ class DotNetSamProjectGenerator(
     )
 
     private val generator = SamProjectGenerator()
-    private val samPanel = SamInitSelectionPanel(generator)
+    private val samPanel = SamInitSelectionPanel(generator) { it.toLowerCase().startsWith("dotnetcore") }
```

Comment:
Query the `RuntmieGroup.DOTNET` so we dont have this logic that is different than elsewhere

Reply:
Good idea, fixed